statsmodels/tsa/vector_ar/util.py

```diff
@@ -241,6 +242,11 @@ def varsim(coefs, intercept, sig_u, steps=100, initvalues=None, seed=None):
         result[p:] += ugen[p:]
     else:
         result[p:] = ugen[p:]
+        
+    if initvalues is not None:
+        if not initvalues.shape == (p, k):
```

Comment:
Should handle case where initvalues are either (k,) or (p,k). In either case I think you only need to modify the docstring and broadcasting will take care of assignment expansion.